lightning/lightning-client.md

```diff
@@ -103,7 +103,8 @@ $ wget https://github.com/lightningnetwork/lnd/releases/download/v$VERSION-beta/
 **Example** of expected output:
 
 ```
-> lnd-linux-amd64-v0.16.3-beta.tar.gz: OK
```

Comment:
Delete this change, comment https://github.com/minibolt-guide/minibolt/pull/70/files#r1257216011